src/components/DressListItem/DressListItem.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import styled from "styled-components";
+import "./DressListItem.css";
+import imageFile1 from "../DressListItem/DressListItemAssets/seraphina.png";
+
+export default () => (
+  <div className="one-item">
+    <div className="dress-list-item-container">
+      <img src={imageFile1} alt="image" />
```

Comment:
Load your image and the dress name as props instead of the actual file and the actual string.